GetIntoTeachingApi/Models/MappingInfo.cs

```diff
@@ -13,6 +13,10 @@ public class MappingInfo
         public IDictionary<string, IDictionary<string, string>> Relationships { get; set; } =
             new Dictionary<string, IDictionary<string, string>>();
 
+        public MappingInfo()

```

Comment:
Is this needed?

Reply:
Yeah you need a parameterless constructor to be able to deserialise into the object

Reply:
ok cool